### DISCUSSION TITLE
Update Wezterm

### DISCUSSION THREAD

**Discussion by Ankhas** [wezterm/wezterm#1028] [Q&A] [answered]
Hello Wezterm community!

I would like to know if there is a proper way for doing the last Wezterm update on my Linux PC. I can't find any documentation on the website or in Github.

**Comment by SuperSandro2000** [comment, marked as answer]
Download the latest archive from the releases and install it like you did the first time.

**Reply by wez** [reply]
Running the setup.exe installer for the newer release will do the right thing.  If you installed from the zip file, you can repeat the same steps you used previously.

**Reply by Ankhas** [reply]
Ok, thanks! 

I think we should add a section on documentation to explain that. Can I create an issue from this discussion?

**Reply by wez** [reply]
If you'd like to submit a PR for the docs, you can click the pen/pencil icon on https://github.com/wez/wezterm/blob/main/docs/install/windows.markdown and go from there! :-)
